xl2times/transforms.py

```diff
@@ -2109,40 +2144,19 @@ def process_uc_wildcards(
 ) -> Dict[str, DataFrame]:
     tag = datatypes.Tag.uc_t
 
-    def make_str(df):
-        if df is not None and len(df) != 0:
-            list_from_df = df.iloc[:, 0].unique()
-            return ",".join(list_from_df)
-        else:
-            return None
-
-    if tag in tables:
+    if tag in tqdm(tables, desc=f"Processing uc_wildcards on tables"):
         start_time = time.time()
         df = tables[tag]
-        query_columns = config.query_columns[tag]
```

Comment:
Why are we not keeping it table type specific here?

Reply:
Another merge hiccup, fixed now.